Kudu.Core/Infrastructure/OperationClient.cs

```diff
@@ -0,0 +1,52 @@
+using System;
+using System.Diagnostics;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Reflection;
+using System.Threading.Tasks;
+using Kudu.Contracts.Tracing;
+using Kudu.Core.Tracing;
+
+namespace Kudu.Core.Infrastructure
+{
+    public class OperationClient
+    {
+        private static Lazy<ProductInfoHeaderValue> _userAgent = new Lazy<ProductInfoHeaderValue>(() =>
+        {
+            var assembly = Assembly.GetExecutingAssembly();
+            var fvi = FileVersionInfo.GetVersionInfo(assembly.Location);
+            return new ProductInfoHeaderValue("kudu", fvi.FileVersion);
+        });
+
+        private readonly ITracer _tracer;
+
+        public OperationClient(ITracer tracer)
+        {
+            _tracer = tracer;
+        }
+
+        public async Task<HttpResponseMessage> PostAsync(string path)
+        {
+            return await PostAsync<string>(path);
+        }
+
+        public async Task<HttpResponseMessage> PostAsync<T>(string path, T content = default(T))
+        {
+            var jwt = System.Environment.GetEnvironmentVariable(Constants.X_MS_SITE_RESTRICTED_JWT);
+            var host = System.Environment.GetEnvironmentVariable(Constants.HTTP_HOST);
+            using (_tracer.Step("POST " + path))
+            {
+                using (var client = new HttpClient())
```

Comment:
i think this will send request just fine against PROD server. but for private stamp, we will get SSL error. shall we ignore SSL error by default?

````
using (var handler = new WebRequestHandler())
{
    handler.ServerCertificateValidationCallback += (sender, cert, chain, sslPolicyErrors) => true;
    ....
}
````